docs/ref/utils.txt

```diff
@@ -539,14 +539,14 @@ https://web.archive.org/web/20110718035220/http://diveintomark.org/archives/2004
 
         from django.utils.functional import keep_lazy, keep_lazy_text
 
-        def fancy_utility_function(s, ...):
+        def fancy_utility_function(s):
```

Comment:
Without this change I get this error:

```
ref/utils.txt:538: code block parse error Cannot parse: 3:30: def fancy_utility_function(s, ...):
```

If there is a way to do it while keeping the `...`, or somehow indicate more parameters can be passed, please let me know.